src/Core/Label.js

```diff
@@ -0,0 +1,146 @@
+import * as THREE from 'three';
+import DEMUtils from 'Utils/DEMUtils';
+import Coordinates from 'Core/Geographic/Coordinates';
+
+const coord = new Coordinates('EPSG:4326');
+let rect;
+
+// set it once
+let STYLE_TRANSFORM = '';
+if (document.documentElement.style.transform !== undefined) {
+    STYLE_TRANSFORM = 'transform';
+} else if (document.documentElement.style.webkitTransform !== undefined) {
+    STYLE_TRANSFORM = 'webkitTransform';
+} else if (document.documentElement.style.mozTransform !== undefined) {
+    STYLE_TRANSFORM = 'mozTransform';
+} else if (document.documentElement.style.oTransform !== undefined) {
+    STYLE_TRANSFORM = 'oTransform';
+} else {
+    STYLE_TRANSFORM = 'transform';
+}
+
+/**
+ * An object that handles the display of a text and/or an icon, linked to a 3D
+ * position. The content of the `Label` is managed through a DOM object, in a
+ * `<div>` handled by the `Label2DRenderer`.
+ *
+ * @property {boolean} isLabel - Used to checkout whether this object is a
+ * Label. Default is true. You should not change this, as it is used internally
+ * for optimisation.
+ * @property {Element} content - The DOM object that contains the content of the
+ * label. The style and the position are applied on this object.
+ * @property {THREE.Vector3} position - The position in the 3D world of the
+ * label.
+ * @property {Coordinates} coordinates - The coordinates of the label.
+ */
+class Label extends THREE.Object3D {
+    /**
+     * @param {Element|string} content - The content; can be a
+     * string, with or without HTML tags in it, or it can be an Element.
+     * @param {Coordinates} coordinates - The world coordinates, where to place
```

Comment:
What is the coordinates projection? it's the same of `view.referenceCrs`?

Reply:
Not necessary, it can be anything, as `label.position` is not affected by the coordinates projection.